src/test/java/ibm/jceplus/junit/openjceplusfips/TestAll.java

```diff
@@ -40,16 +36,6 @@
         TestSHA3_224.class, TestSHA3_256.class, TestSHA3_384.class, TestSHA3_512.class,
         TestRSAKeyInterop.class, TestRSAKeyInteropBC.class, TestRSAPSS2.class,
         TestFIPSVerifyOnlyTest.class, TestRSASignatureWithSpecificSize.class})
-
+@Suite
 public class TestAll {
-
-    public static Test suite() {
-        return new JUnit4TestAdapter(TestAll.class);
-    }
-
-    public static void main(String[] args) {
-        junit.textui.TestRunner.run(suite());
-    }
-
-}
-
+}
```

Comment:
Add new line.

Reply:
New line added in commit bd27b562b46b98c872982261f4a7cb8bb2145955

Additionally a checkstyle rule was added to enforce this in the future.